tests/expectations/test_expectation_atomic_renderers.py

```diff
@@ -178,6 +178,14 @@ def test_atomic_prescriptive_summary_expect_column_kl_divergence_to_be_less_than
 
     res = rendered_content.to_json_dict()
     pprint(res)
+
+    # replace version of vega-lite in res to match snapshot test
+    current_graph_string = res["value"]["graph"]
+    start_index = current_graph_string.index("vega-lite/") + len("vega-lite/")
+    end_index = current_graph_string.index(".json")
+    modified_graph_string = current_graph_string[:start_index] + "v4.8.1" + current_graph_string[end_index:]
+    res["value"]["graph"] = modified_graph_string
```

Comment:
```python
current_graph_string = res["value"]["graph"]
modified_graph_string = re.sub(r"v\d*\.\d*\.\d*", "v4.8.1", current_graph_string)
res["value"]["graph"] = modified_graph_string
```

Reply:
@talagluck I found a possibly solution using regex if that helps clean things up!